src/hb-ot-os2-table.hh

```diff
@@ -216,8 +216,10 @@ struct OS2
 					  uint16_t *min_cp, /* OUT */
 					  uint16_t *max_cp  /* OUT */)
   {
-    *min_cp = codepoints->get_min ();
-    *max_cp = codepoints->get_max ();
+    hb_codepoint_t min_codepoint = codepoints->get_min ();
+    hb_codepoint_t max_codepoint = codepoints->get_max ();
+    *min_cp = min_codepoint > 0xFFFF ? 0xFFFF : min_codepoint;
```

Comment:
could also write this as:
*min_cp = hb_min(0xFFFF, codepoints->get_min());

Likewise for max_cp